doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_smp.rst

```diff
@@ -0,0 +1,460 @@
++-----------------------------+-------------------------+----------------------------------+
+| State information           | Embedded within the     | Update candidate existence is    |
+|                             | image slot metadata     | represented within the SUIT      |
+|                             |                         | manifest.                        |
++-----------------------------+-------------------------+----------------------------------+
+
+SUIT in SMP image management group
+==================================
+
+The SMP protocol can provide SMP-based SUIT image management, assuming that even a complex SUIT-based system can be represented as a *single image* system.
+In this model, the contents of the *primary slot* (``image 0``) represent the currently installed SUIT root manifest, while the *secondary slot* holds the update candidate root manifest.
+
+Get state of images request
+---------------------------
+
+This command retrieves images and their states.
+The response is in CBOR format:
+
+.. code-block::
+
+    {
+    (str)"images" : [
+        {
+            (str,opt)"image"      : (int)       //SUIT - always 0.
+            (str)"slot"           : (int)       //SUIT - 0 - "primary" - installed Root manifest, 1 - update candidate manifest in the DFU partition.
+            (str)"version"        : (str)       //SUIT - sequence-number from root manifest.
+            (str)"hash"           : (byte str)  //SUIT - digest of root manifest.
+        }
+        ...
+    ]}
+
+Image upload request
+--------------------
+
+This command uploads an image containing a SUIT envelope to be stored in the DFU partition or a dump of the DFU cache partition.
+The CBOR data format is the following:
+
+.. code-block::
+
+    {
+        (str,opt)"image"    : (uint)     //SUIT - 0 - DFU partition, 1 to n - DFU cache partition 0 to n-1.
+        (str,opt)"len"      : (uint)
+        (str)"off"          : (uint)
+        (str,opt)"data"     : (byte str)
+    }
+
+Set state of images request
+---------------------------
+
+This command triggers an installation process using the SUIT envelope stored in the DFU partition.
+The CBOR data format is the following:
+
+.. code-block:: cbor
+
+    {
+        (str)"confirm" : (bool) //Must be set to "true".
+    }
+
+SUIT SMP protocol extension
+===========================
+
+The existing image management approach works well for basic scenarios involving single-step SUIT envelope pushes.
+More advanced scenarios require defining a new SMP management group.
+
+Management Group ID
+-------------------
+
+The new SUIT SMP protocol uses ``MGMT_GROUP_ID_PERUSER + 2`` (for example, 66).
+
+Commands
+--------
+
+Get SUIT manifests list request
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This command allows retrieving information about the roles of manifests supported by the device::
+
+    OP: MGMT_OP_READ (0), MGMT_OP_READ_RSP(1)
+
+    Command ID: 0
```

Comment:
Just noticed this has the same information, but formatted differently. Either format is fine with me.